03-task-manager/starter/app.js

```diff
@@ -3,16 +3,20 @@ const app = express()
 const tasks = require('./routes/tasks')
 const connectDB = require('./db/connect')
 require('dotenv').config()
+const notFound = require('./middleware/not-found')
+const errorHandlerMiddleware = require('./middleware/errorHandlerMiddleware')
```

Comment:
Your `errorHandlingMiddleware` import is incorrect.
```suggestion
const errorHandlerMiddleware = require('./middleware/error-handler')
```